packages/test-app/README.md

```diff
@@ -2,6 +2,17 @@
 
 This apps intended purpose is to quickly test the functionality of the EB when developing locally. There is also a deployed instance of the app that can be used to test EB in a production.
 
+## How to Start
+
+1. Create a `.env` in this package with the environment variables described in chapter [Config](#config).
+2. Make sure that the preview URL of the experience type is set to `http://localhost:5173/{entry.fields.slug}`.
```

Comment:
Step 2 isn't necessary, the visual editor now uses the experience selected in the editor instead of relying on the app to fetch it.

Reply:
You mean it's not necessary to add the slug, right? Still we need to point to `localhost:5173` :)

Reply:
I guess it will still be necessary to render test-app in preview mode.